EasyRestProjectNetTeam2/EasyRestPages/MenuPage.cs

```diff
@@ -6,8 +6,8 @@ namespace EasyRestProjectNetTeam2.EasyRestPages
 {
     public class MenuPage : BasePage
     {
-        public DatePickerComponent DatePickerComponent { get; }
-        public TimePickerComponent TimePickerComponent { get; }
+        public DatePickerComponent DatePickerComponent { get; set; }
```

Comment:
why did you add public setters?

Reply:
because we caught errors

Reply:
you can't have a public setter because you can re-initialize the component on the test level at any moment, please change the access modifier